runtime/query.go

```diff
@@ -4,26 +4,21 @@ import (
 	"context"
 	"fmt"
 	"strings"
-
-	"github.com/rilldata/rill/runtime/pkg/observability"
-	"go.opentelemetry.io/otel/metric/global"
 )
 
-var (
-	meter                   = global.Meter("runtime")
-	queryCacheHitsCounter   = observability.Must(meter.Int64Counter("query_cache.hits"))
-	queryCacheMissesCounter = observability.Must(meter.Int64Counter("query_cache.misses"))
-)
+type CacheObject struct {
+	Result      any
+	SizeInBytes int64
+}
```

Comment:
It relates to `Query`, not the cache, so maybe call it `QueryResult`.

Then `Result any` can just be `Value any` and `SizeInBytes` can just be `Bytes int64`